packages/horizon-householder-appeal-publish/handler-reply.js

```diff
@@ -58,6 +58,8 @@ const populateDocuments = (body) => {
     }
   });
 
+  documents.push({ id: body.submission.pdfStatement.uploadedFile.id, type: sectionTypes.pdfType });
```

Comment:
What happens here if there is no pdfStatement in the submission on the body? The FWA stuff will break as it doesnt always have a body.submission.pdfStatement object.....

Reply:
I wasn't aware that there would ever not be a pdfStatement. Will amend.

Reply:
I mean, i don't know for certain that it is or is not possible, just seems like a safety check that doesnt cost us anything

Reply:
If that's the case the process should have failed before now if it was unable to create a pdf and place it in this location. The whole process should stop, and this would be an innappropriate place for it to fail.

Reply:
"should have" - but can this service, in isolation, guarantee that it will never get a body.submission without a pdfStatement on it? And if for some reason it does get sent something without a pdfStatement, will we get a useful error message out that quickly shows us exactly why it is failing, or will it crash out, never returning the documents object that its supposed to, leaving us digging around trying to work out why a set of LPA documents isn't in Horizon.

Reply:
I think that's fair. We probably need to have a conversation about what happens if this occurs. Will add the error messge.